File: openapi/openapi.json/code_samples/typescript/iot-location_v1_locationCheck/post.ts
import { DeviceLocationRequest, TestSdk } from 'test-sdk';

(async () => {
  const testSdk = new TestSdk({
    token: 'YOUR_TOKEN',
  });

  const horizontalPrecision = HorizontalPrecision.AREA;

  const deviceLocationRequest: DeviceLocationRequest = {
    iccid: '8901882024109918312',
    horizontalPrecision: horizontalPrecision,
    respTimeOut: 30,
    maxCacheAge: 300,
  };

  const { data } = await testSdk.deviceLocation.locationCheck(input);

  console.log(data);
})();

File: openapi/code_samples/python/iot-location_v1_locationCheck/post.py
from test_sdk import TestSdk, Environment
from test_sdk.models import DeviceLocationRequest

sdk = TestSdk(
    access_token="YOUR_ACCESS_TOKEN",
    base_url=Environment.DEFAULT.value,
    timeout=10000
)

request_body = DeviceLocationRequest(
    iccid="8901882024109918312",
    horizontal_precision="AREA",
    resp_time_out=30,
    max_cache_age=300
)

result = sdk.device_location.location_check(request_body=request_body)

print(result)